src/main/charts/confluence/templates/_fluentd_templates.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "fluentd.container" -}}
+{{ if .Values.fluentd.enabled }}
+- name: fluentd
+  image: {{ .Values.fluentd.imageName }}
+  command: ["fluentd", "-c", "/fluentd/etc/fluent.conf", "-v"]
+  volumeMounts:
+    - name: local-home
+      mountPath: /application-data
+      readOnly: true
+    - name: fluentd-config
+      mountPath: /fluentd/etc
+      readOnly: true
+  env:
+    - name: POD_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.name
+    - name: POD_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    - name: POD_IP
+      valueFrom:
+        fieldRef:
+          fieldPath: status.podIP
+    - name: HELM_RELEASE_NAME
+      value: {{ include "confluence.fullname" . }}
```

Comment:
These env vars are injected into the sidecar so that fluentd can add them as log record fields